datasets/xfun/README.md

```diff
@@ -0,0 +1,211 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- expert-generated
+languages:
+- zh
+- de
+- es
+- fr
+- it
+- ja
+- pt
+licenses:
+- cc-by-nc-sa-4.0
+multilinguality:
+- multilingual
+pretty_name: XFUN
+size_categories:
+- unknown
+source_datasets:
+- original
+task_categories:
+- text-classification
+task_ids:
+- entity-linking-classification
+---
+
+# Dataset Card for XFUN
+
+## Table of Contents
+- [Table of Contents](#table-of-contents)
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage: https://github.com/doc-analysis/XFUND**
+- **Repository: https://github.com/doc-analysis/XFUND**
+- **Paper: https://arxiv.org/abs/2104.08836**
+- **Leaderboard: https://github.com/doc-analysis/XFUND#results**
+- **Point of Contact: Lei Cui (`lecu@microsoft.com`) or Furu Wei (`fuwei@microsoft.com`)**
+
+### Dataset Summary
+
+XFUND is a multilingual form understanding benchmark dataset that includes human-labeled forms with key-value pairs in 7 languages (Chinese, Japanese, Spanish, French, Italian, German, Portuguese).
+
+### Supported Tasks and Leaderboards
+
+1. Semantic Entity Recognitionm
+2. Key-Value Pair Relation-Extraction
+
+### Languages
+
+Chinese, Japanese, Spanish, French, Italian, German, Portuguese
+
+## Dataset Structure
+
+### Data Instances
+
+There are seven configurations in this dataset, which are `xfun.zh`, `xfun.ja`, `xfun.es`, `xfun.fr`, `xfun.it`, `xfun.de`, and `xfun.pt`.
+
+An example of `xfun.fr` looks as follows:
+
+```
+DatasetDict({
+    train: Dataset({
+        features: ['id', 'input_ids', 'bbox', 'labels', 'image', 'entities', 'relations'],
+        num_rows: 202
+    })
+    validation: Dataset({
+        features: ['id', 'input_ids', 'bbox', 'labels', 'image', 'entities', 'relations'],
+        num_rows: 71
+    })
+})
+```
```

Comment:
The example will be very verbose and become a long passage because of the length of `input_ids`, `labels`, ...etc.:
```
{'id': 'fr_train_0_0', 'input_ids': [72791, 2149, 104, 25, 101287, 990, 106, 22364, 61038, 35213, 1514, 335, 76783, 11338, 109893, 122112, 84780, 31244, 51552, 441, 132824, 35213, 14614, 167795, 618, 89097, 85621, 7, 62, 5, 61038, 35213, 238530, 40713, 839, 80020, 615, 55448, 628, 38016, 149896, 13, 8, 21, 17752, 22, 6, 102250, 115, 19183, 94228, 531, 104067, 12, 438, 39, 6470, 68915, 170158, 6, 58745, 14154, 60221, 166132, 4, 436, 187367, 17166, 7, 628, 6, 138971, 64, 54601, 1830, 628, 123708, 64, 166697, 13, 628, 5054, 9942, 384, 50763, 26551, 132, 7, 16, 6, 58745, 4880, 192508, 196417, 3768, 72813, 25388, 148573, 15, 112, 96, 25, 165264, 16, 6, 191712, 981, 84624, 5, 277, 20897, 64, 88154, 64, 22469, 38016, 48218, 1426, 5, 191712, 5, 277, 787, 2041, 2312, 80273, 13, 104, 25, 40098, 12, 15, 16305, 27227, 104, 25, 309, 1193, 21, 108090, 42518, 1236, 253, 2771, 49894, 16, 6, 166697, 13, 20, 6, 179149, 8, 5054, 6, 136711, 55448, 11129, 628, 60013, 916, 44386, 6, 136711, 14400, 93781, 253, 1284, 123107, 3190, 6, 136711, 56187, 83221, 8, 45556, 7, 20, 127847, 7, 8, 5054, 6, 136711, 113800, 40567, 628, 29096, 6, 136711, 113800, 29208, 6, 136711, 6, 166697, 11, 76659, 16867, 26988, 6, 136711, 52236, 107, 108090, 15, 7, 11870, 238, 85789, 16, 6, 136711, 1031, 853, 211161, 1236, 179621, 8, 96, 25, 40098, 12, 438, 39, 167599, 71034, 10763, 36166, 58503, 44386, 384, 50763, 26551, 4880, 169842, 122381, 1530, 72813, 25388, 452, 6827, 981, 84624, 5, 277, 1398, 15270, 86, 28022, 115, 19183, 94228, 531, 104067, 12, 438, 39, 24479, 171799, 2312, 628, 32348, 803, 4273, 238, 19077, 384, 50763, 26551, 4880, 19051, 304, 151616, 3117, 72813, 25388, 145010, 647, 981, 84624, 5, 277], 'bbox': [[630, 128, 721, 141], [630, 128, 721, 141], [724, 129, 826, 141], [724, 129, 826, 141], [724, 129, 826, 141], [867, 128, 876, 139], [794, 141, 824, 152], [794, 141, 824, 152], [80, 158, 236, 175], [80, 158, 236, 175], [245, 158, 276, 173], [281, 158, 343, 173], [281, 158, 343, 173], [78, 177, 254, 194], [78, 177, 254, 194], [78, 177, 254, 194], [78, 177, 254, 194], [261, 177, 288, 194], [292, 177, 443, 194], [292, 177, 443, 194], [292, 177, 443, 194], [292, 177, 443, 194], [449, 177, 481, 194], [488, 177, 578, 194], [488, 177, 578, 194], [75, 238, 310, 253], [335, 237, 511, 253], [335, 237, 511, 253], [85, 267, 101, 280], [85, 267, 101, 280], [103, 267, 227, 280], [103, 267, 227, 280], [233, 267, 400, 279], [233, 267, 400, 279], [233, 267, 400, 279], [233, 267, 400, 279], [75, 297, 95, 311], [98, 297, 216, 312], [220, 297, 243, 312], [247, 297, 339, 312], [342, 297, 416, 312], [342, 297, 416, 312], [419, 297, 441, 312], [445, 297, 462, 312], [465, 297, 509, 313], [512, 297, 534, 313], [538, 297, 538, 297], [538, 297, 601, 313], [605, 297, 628, 313], [631, 297, 687, 313], [690, 297, 768, 313], [772, 298, 795, 313], [799, 298, 884, 313], [799, 298, 884, 313], [75, 325, 109, 338], [75, 325, 109, 338], [197, 327, 246, 338], [75, 350, 132, 363], [135, 349, 189, 363], [192, 349, 192, 349], [192, 349, 201, 363], [239, 347, 253, 360], [256, 348, 279, 360], [283, 348, 326, 360], [283, 348, 326, 360], [329, 347, 381, 360], [329, 347, 381, 360], [75, 373, 105, 388], [75, 373, 105, 388], [108, 373, 127, 388], [131, 373, 131, 373], [131, 373, 215, 388], [131, 373, 215, 388], [131, 373, 215, 388], [131, 373, 215, 388], [219, 373, 239, 388], [243, 373, 413, 388], [243, 373, 413, 388], [243, 373, 413, 388], [243, 373, 413, 388], [415, 374, 437, 388], [439, 374, 472, 388], [525, 375, 574, 388], [75, 423, 168, 439], [75, 423, 168, 439], [75, 423, 168, 439], [75, 423, 168, 439], [75, 423, 168, 439], [75, 423, 168, 439], [171, 424, 171, 424], [171, 424, 183, 440], [220, 426, 303, 437], [220, 426, 303, 437], [220, 426, 303, 437], [220, 426, 303, 437], [433, 423, 495, 438], [433, 423, 495, 438], [499, 423, 585, 438], [588, 423, 610, 438], [588, 423, 610, 438], [614, 423, 704, 438], [614, 423, 704, 438], [614, 423, 704, 438], [614, 423, 704, 438], [724, 425, 724, 425], [724, 425, 869, 438], [724, 425, 869, 438], [724, 425, 869, 438], [724, 425, 869, 438], [724, 425, 869, 438], [76, 448, 200, 462], [76, 448, 200, 462], [76, 448, 200, 462], [76, 448, 200, 462], [76, 448, 200, 462], [203, 448, 310, 462], [203, 448, 310, 462], [374, 452, 498, 462], [374, 452, 498, 462], [374, 452, 498, 462], [374, 452, 498, 462], [374, 452, 498, 462], [74, 487, 93, 500], [102, 488, 193, 502], [102, 488, 193, 502], [102, 488, 193, 502], [102, 488, 193, 502], [195, 487, 321, 502], [195, 487, 321, 502], [195, 487, 321, 502], [322, 488, 329, 501], [96, 502, 156, 514], [96, 502, 156, 514], [96, 502, 156, 514], [158, 502, 187, 514], [158, 502, 187, 514], [158, 502, 187, 514], [189, 502, 200, 514], [204, 502, 216, 514], [218, 502, 278, 514], [281, 502, 374, 514], [281, 502, 374, 514], [377, 502, 385, 514], [389, 502, 421, 514], [424, 502, 508, 514], [424, 502, 508, 514], [75, 535, 75, 535], [75, 535, 150, 549], [75, 535, 150, 549], [153, 535, 159, 548], [162, 535, 162, 535], [162, 535, 206, 549], [209, 535, 229, 549], [231, 535, 262, 549], [265, 526, 265, 526], [265, 526, 302, 552], [331, 535, 423, 549], [427, 535, 474, 548], [477, 535, 497, 548], [501, 535, 613, 548], [501, 535, 613, 548], [501, 535, 613, 548], [616, 526, 616, 526], [616, 526, 655, 552], [691, 536, 762, 549], [691, 536, 762, 549], [765, 536, 774, 549], [777, 536, 803, 549], [806, 535, 858, 549], [806, 535, 858, 549], [868, 526, 868, 526], [868, 526, 904, 553], [75, 567, 154, 581], [158, 567, 209, 581], [213, 567, 230, 581], [234, 567, 311, 581], [234, 567, 311, 581], [314, 567, 321, 580], [325, 566, 370, 580], [325, 566, 370, 580], [373, 566, 392, 580], [395, 566, 426, 580], [429, 557, 429, 557], [429, 557, 468, 582], [478, 567, 516, 579], [521, 567, 587, 580], [590, 567, 610, 580], [613, 567, 656, 580], [662, 557, 662, 557], [662, 557, 701, 585], [721, 568, 758, 580], [764, 568, 861, 580], [868, 557, 868, 557], [868, 557, 904, 585], [75, 599, 75, 599], [75, 599, 169, 612], [75, 599, 169, 612], [75, 599, 169, 612], [172, 600, 245, 613], [172, 600, 245, 613], [246, 590, 246, 590], [246, 590, 282, 615], [331, 598, 372, 612], [331, 598, 372, 612], [375, 598, 441, 612], [445, 598, 514, 612], [445, 598, 514, 612], [445, 598, 514, 612], [445, 598, 514, 612], [445, 598, 514, 612], [445, 598, 514, 612], [522, 589, 522, 589], [522, 589, 560, 617], [76, 636, 93, 651], [97, 636, 220, 652], [97, 636, 220, 652], [97, 636, 220, 652], [223, 636, 267, 652], [270, 636, 292, 652], [295, 636, 426, 652], [295, 636, 426, 652], [295, 636, 426, 652], [295, 636, 426, 652], [72, 672, 105, 685], [72, 672, 105, 685], [238, 671, 283, 683], [73, 696, 136, 709], [73, 696, 136, 709], [236, 694, 311, 708], [236, 694, 311, 708], [236, 694, 311, 708], [73, 720, 147, 734], [73, 720, 147, 734], [73, 720, 147, 734], [235, 721, 327, 734], [235, 721, 327, 734], [235, 721, 327, 734], [235, 721, 327, 734], [73, 746, 139, 759], [73, 746, 139, 759], [238, 747, 384, 761], [238, 747, 384, 761], [238, 747, 384, 761], [238, 747, 384, 761], [238, 747, 384, 761], [238, 747, 384, 761], [74, 783, 91, 798], [94, 783, 181, 799], [94, 783, 181, 799], [184, 783, 291, 800], [294, 783, 318, 800], [321, 783, 379, 800], [381, 783, 462, 800], [465, 784, 488, 799], [491, 784, 579, 799], [491, 784, 579, 799], [72, 807, 106, 821], [72, 807, 106, 821], [263, 809, 306, 820], [72, 832, 173, 846], [72, 832, 173, 846], [177, 832, 196, 846], [200, 832, 262, 846], [345, 830, 418, 842], [345, 830, 418, 842], [345, 830, 418, 842], [345, 830, 418, 842], [73, 856, 152, 872], [73, 856, 152, 872], [73, 856, 152, 872], [276, 861, 362, 873], [276, 861, 362, 873], [276, 861, 362, 873], [276, 861, 362, 873], [276, 861, 362, 873], [71, 881, 139, 896], [71, 881, 139, 896], [269, 885, 420, 897], [269, 885, 420, 897], [269, 885, 420, 897], [269, 885, 420, 897], [269, 885, 420, 897], [269, 885, 420, 897]], 'labels': [0, 0, 0, 0, 0, 0, 0, 0, 3, 6, 6, 6, 6, 3, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 6, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 5, 2, 1, 5, 5, 5, 2, 4, 4, 4, 4, 4, 1, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 2, 1, 5, 5, 5, 5, 5, 5, 5, 2, 4, 4, 4, 1, 5, 5, 5, 5, 5, 5, 5, 5, 2, 4, 4, 4, 4, 4, 1, 5, 5, 5, 5, 5, 5, 2, 4, 4, 4, 4, 1, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 5, 2, 4, 4, 4, 4, 4, 4, 4, 4, 4, 2, 4, 4, 4, 4, 4, 4, 4, 2, 4, 4, 4, 4, 4, 4, 4, 2, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 2, 4, 4, 4, 4, 4, 2, 4, 4, 4, 2, 4, 4, 4, 4, 4, 4, 4, 2, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 5, 2, 1, 5, 2, 4, 4, 1, 5, 5, 2, 4, 4, 4, 1, 5, 2, 4, 4, 4, 4, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 5, 2, 1, 5, 5, 5, 2, 4, 4, 4, 1, 5, 5, 2, 4, 4, 4, 4, 1, 5, 2, 4, 4, 4, 4, 4], 'entities': {'start': [8, 13, 54, 56, 57, 61, 67, 82, 83, 91, 95, 104, 110, 117, 122, 146, 156, 164, 172, 184, 190, 194, 202, 223, 225, 226, 228, 231, 234, 238, 240, 256, 258, 259, 263, 267, 270, 275, 277], 'end': [13, 25, 56, 57, 61, 67, 82, 83, 91, 95, 104, 110, 117, 122, 146, 156, 164, 172, 184, 190, 194, 202, 213, 225, 226, 228, 231, 234, 238, 240, 246, 258, 259, 263, 267, 270, 275, 277, 283], 'label': [0, 0, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 2, 2, 2, 2, 2, 2, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2, 1, 2]}, 'relations': {'head': [2, 4, 6, 8, 10, 12, 14, 14, 14, 14, 14, 14, 14, 14, 23, 25, 27, 29, 31, 33, 35, 37], 'tail': [3, 5, 7, 9, 11, 13, 15, 16, 21, 20, 22, 17, 18, 19, 24, 26, 28, 30, 32, 34, 36, 38], 'start_index': [54, 57, 67, 83, 95, 110, 122, 122, 122, 122, 122, 122, 122, 122, 223, 226, 231, 238, 256, 259, 267, 275], 'end_index': [57, 67, 83, 95, 110, 122, 156, 164, 202, 194, 213, 172, 184, 190, 226, 231, 238, 246, 259, 267, 275, 283]}}
```